### PR TITLE
fix #160 and #157

### LINKED ISSUE #157: Display total value of TX on TX detail page
## Story 
As a user, I want to view the total value of NEAR sent in a transaction, so I can determine whether or not a deposit or transfer was included with any actions in the transaction.

## Acceptance Criteria

TX with single `functionCall` action ([example](https://explorer.nearprotocol.com/transactions/ARwKeUACYbu9PrTSJ8L7CR9PfbJou4xgmWbF9yhBS7mL))
- [ ] Value is displayed correctly when `deposit` > 0 
- [ ] Value is displayed as 0 when `deposit` = 0 

TX with single `transfer` action ([example](https://explorer.nearprotocol.com/transactions/niKQWurcqHT7pxMyt8PCj8v9i3eZwH4hmRBXVL2DU4c))
- [ ] Value is displayed correctly

TX with multiple actions including a `transfer` (e.g. [all calls](https://explorer.nearprotocol.com/accounts/test) by `@test`)
- [ ] Value from `transfer` action is displayed correctly

- [ ] Mobile ok
- [ ] Tested

## Mockup
[Mockup](https://yc3.s3.us-east-2.amazonaws.com/SS/Screen+Shot+2020-01-14+at+10.46.13+AM.png)

### LINKED ISSUE #160: Display FunctionCall details on the Transaction Details page
## Story

As a contract developer who debugs a contract, I want to see the input arguments that are passed into the contract so I can have a better idea of what is happening.

## Acceptance Criteria

- [x] There is a detailed mode of ActionRow component that is capable to display extra information for the given action
- [x] Display FunctionCall method and arguments (just decode the JSON encoded as base64 string provided in `args` field) as extra information on ActionRow
- [x] Mobile ok
- [x] Tested (a unit-test for ActionRow detailed mode displaying FunctionCall action)

## Notes

@vgrichina:
> probably worth checking if it's valid UTF-8 string and parses as JSON, and if it's not – do hexdump instead (e.g. using smth like this https://www.npmjs.com/package/hexdump-nodejs)

@nearmax:
> FYI as discussed in a different thread. JSON args might not be staying for long. It is very likely we will be switching to Borsh and front-end will be retrieving args schema from metadata() function or somesuch.

### PR DESCRIPTION
**Pull request by icerove**
1. fix the #160 
2. fix the issue for account create time and hash when account is from genesis (remaining query block 0 to fetch data)
3. fix #157

### CODE REVIEW

**Comment by frol** [comment]
Also, please, rebase on the current master branch where I have slightly updated the CI, so it should be green (I have disabled the backend testing for now, since we don't have tests there, and it was just masked the failing frontend tests).

File: frontend/src/pages/accounts/[id].jsx [hunk 1]
```diff
@@ -18,7 +18,22 @@ export default class extends React.Component {
       return { id, err };
     }
   }
-  
+
+  state = {
+    id: this.props.id,
+    timestamp: 1577836800000,
+    address: "Not disclosed"
```
[the remark this answers is not in the document]
Comment (icerove) [reply frontend/src/pages/accounts/[id].jsx]: for loading, it is okay to say loading and then display the correct information, but when it is from genesis, now we do not have that information. I see you create issue in nearcore, maybe we create an issue for this, too. Now I will just set to loading

File: frontend/src/components/transactions/TransactionDetails.tsx [hunk 2]
```diff
@@ -7,87 +8,140 @@ import AccountLink from "../utils/AccountLink";
...
+        actionKind = Object.keys(action)[0] as keyof T.Action;
+        actionArgs = action[actionKind] as T.Action;
+      }
+      if (actionKind === "Transfer" || actionKind === "FunctionCall") {
+        const args = JSON.stringify(actionArgs);
+        const index = args.search("deposit") + 10;
+        const pre_deposit = args.slice(index);
+        const index_d = pre_deposit.indexOf('"');
+        const deposit = pre_deposit.slice(0, index_d);
+        this.setState({ deposit });
```
[the remark this answers is not in the document]
Comment (icerove) [reply frontend/src/components/transactions/TransactionDetails.tsx]: can not and do not know why 😢

[the remark this answers is not in the document]
Comment (icerove) [reply frontend/src/components/transactions/TransactionDetails.tsx]: change actionArgs type to any and have a look here

File: frontend/src/components/transactions/TransactionDetails.tsx [hunk 3]
```diff
@@ -7,87 +8,140 @@ import AccountLink from "../utils/AccountLink";
...
-        }
+  render() {
+    const { transaction } = this.props;
+    const { deposit } = this.state;
+    let depositShow;
+    if (deposit === "0") {
+      depositShow = "0 Ⓝ";
+    } else {
+      depositShow = <Balance amount={deposit} />;
+    }
```
[the remark this answers is not in the document]
Comment (icerove) [reply frontend/src/components/transactions/TransactionDetails.tsx]: it does not have, now I fix this in Balance.js

File: frontend/src/pages/accounts/[id].jsx [hunk 4]
```diff
@@ -18,7 +18,22 @@ export default class extends React.Component {
       return { id, err };
     }
   }
-  
+
+  state = {
+    id: this.props.id,
```
[the remark this answers is not in the document]
Comment (icerove) [reply frontend/src/pages/accounts/[id].jsx]: deleted

File: frontend/src/pages/accounts/[id].jsx [hunk 5]
```diff
@@ -29,7 +44,7 @@ export default class extends React.Component {
           {this.props.err ? (
             `Information is not available at the moment. Please, check if the account name is correct or try later.`
           ) : (
-            <AccountDetails account={this.props}/>
+            <AccountDetails account={this.props} accountBasic={this.state} />
```
[the remark this answers is not in the document]
Comment (icerove) [reply frontend/src/pages/accounts/[id].jsx]: merged

**Pull request merged**